test/dirStructure/folderTest.js

```diff
@@ -0,0 +1,220 @@
+        done()
+      }
+    })
+  })
+
+  it('should generate "publicFolder" directory', function (done) {
+    const foldertest = path.join(appDir, app.expressApp.get('params').publicFolder)
+    fs.lstat(foldertest, (err, stats) => {
+      if (err) {
+        done(err)
+      } else {
+        assert.equal(stats.isDirectory(), true, `"PublicFolder" was not made into a directory successfully`)
+        done()
+      }
+    })
+  })
+
+  it('should generate "js" source directory', function (done) {
+    const foldertest = path.join(appDir, app.expressApp.get('params').js.sourceDir)
+    fs.lstat(foldertest, (err, stats) => {
+      if (err) {
+        done(err)
+      } else {
+        assert.equal(stats.isDirectory(), true, `"js source directory" was not made into a directory successfully`)
+        done()
+      }
+    })
+  })
+
+  it('should generate "css" source directory', function (done) {
+    const foldertest = path.join(appDir, app.expressApp.get('params').css.sourceDir)
+    fs.lstat(foldertest, (err, stats) => {
+      if (err) {
+        done(err)
+      } else {
+        assert.equal(stats.isDirectory(), true, `"css source directory" was not made into a directory successfully`)
+        done()
+      }
+    })
+  })
+
+  it('should generate the "image" directory that was put in "staticsSymlinksToPublic"', function (done) {
+    const foldertest = path.join(appDir, '/staticsRootTest/images')
+    fs.lstat(foldertest, (err, stats) => {
+      if (err) {
+        done(err)
+      } else {
+        assert.equal(stats.isDirectory(), true, `"image" was not made into a directory successfully`)
+        done()
+      }
+    })
+  })
+
+  it('should generate the "image" symlink', function (done) {
+    const filetest = path.join(appDir, '/publicFolderTest/images')
+    fs.lstat(filetest, (err, stats) => {
+      if (err) {
+        done(err)
+      } else {
+        assert.equal(stats.isSymbolicLink(), true, `"image" was not made into a symlink successfully`)
+        done()
+      }
+    })
+  })
+
+  it('should generate the "jsTest" symlink', function (done) {
+    const filetest = path.join(appDir, '/publicFolderTest/jsTest')
+    fs.lstat(filetest, (err, stats) => {
+      if (err) {
+        done(err)
+      } else {
+        assert.equal(stats.isSymbolicLink(), true, `"jsTest" was not made into a symlink successfully`)
+        done()
+      }
+    })
+  })
+
+  it('should generate the "cssTest" symlink', function (done) {
+    const filetest = path.join(appDir, '/publicFolderTest/cssTest')
+    fs.lstat(filetest, (err, stats) => {
+      if (err) {
+        done(err)
+      } else {
+        assert.equal(stats.isSymbolicLink(), true, `"cssTest" was not made into a symlink successfully`)
+        done()
+      }
+    })
+  })
+
+  it('should have the path of cssPath and the path by putting together appDir, staticsRoot param and css.sourceDir param equal', function () {
```

Comment:
Reword these to `should set "cssPath" express variable to absolute path of "css.sourceDir"`